django/contrib/staticfiles/finders.py

```diff
@@ -90,6 +93,13 @@ def check(self, **kwargs):
                     'STATIC_ROOT setting.',
                     id='staticfiles.E002',
                 ))
+            elif not os.path.isdir(root):
+                errors.append(Warning(
+                    f'The directory {root} in the STATICFILES_DIRS setting '
+                    'does not exist.',
+                    hint='Perhaps the directory is not tracked in Git?',
```

Comment:
I don't see much value in this hint, also `Git` is not the only VCS. Please remove it.

Reply:
Yeah, that occurred to me also, but I guess I was trying to address the reporter's concern that beginners will still struggle without a hint. But that's what SO is for. I'm happy to remove it.